docs/getting-started/editor/Pulsar.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Pulsar
+permalink: wiki/Pulsar/
+layout: wiki
+id: Pulsar
+---
+----
+
+![pulsaricon](pulsaricon.png)
+
+[Pulsar](https://pulsar-edit.dev/) is a new code editor that is open-source and community-led. It is based on Atom and was started after the announcement to [Sunset Atom](https://github.blog/2022-06-08-sunsetting-atom/) (Dec 15, 2022). The [stated goal](https://pulsar-edit.dev/about.html) is to "modernize, update and improve the original Atom project into a contemporary, hackable and fully open editor." This includes support for the package repository which provides for community contributions, include our Tidal package.
+
+**Status**: Pulsar is still in Beta. New releases are available regularly. "The releases do seem to be working for most people and use cases." There is a known issue with MacOS performance. "Often times this can be resolved by disabling the `github` package."
+
+---
+**NOTE**: The projected timeline to have Pulsar ready with the Tidalcycles package is the end of 2022. Until then, new Tidal users are encouraged to wait for the new Pulsar editor or select one of the other editors. Tidal users who have a working Atom editor configured with the Tidalcycles package can continue to use it. But post Atom sunsetting, the Atom package manager will stop working. At that point you will no longer be able to use the Atom package manager to install or update the tidalcycles package.
```

Comment:
End of 2022 is very soon. Can we instead of a date refer to a Github issue or discord thread?

Reply:
I added the discord channel info. 
I haven't heard yet from ndr-brt -- but I suggest to go ahead and merge this PR.  Changes can always be done later. 
We will be adding more to this page once Pulsar and the pkg mgt is ready anyway. @Zalastax can you take care of that?

Reply:
Merged! As you say, changes can be done later if and when are necessary.
Amazing work!

Reply:
Thanks both, @HighHarmonics2 we can give you merge privs if you'd like to make some uncontroversial refinements a bit more directly?

Reply:
@yaxu - sure, it would be helpful. I will need to update this page and other pages related to installation and editors.

Reply:
@HighHarmonics2 done!